src/components/PostsWrap.tsx

```diff
@@ -1,17 +1,18 @@
 import React, { useState, useEffect } from 'react';
 import LayoutToggle from './LayoutToggle';
-import Modal from './Modal';
 import SecondaryNav from './SecondaryNav';
 import GridDisplay from './GridDisplay';
 import ListDisplay from './ListDisplay';
 import { fetchRecommendations } from '../lib/supabase';
 import useSupabaseAuth from '../hooks/useSupabaseAuth';
+import PrimaryNav from './PrimaryNav';
+import pizza from '../../public/pizza.svg'
 
 interface PostWrapProps{
   textToSearch: string
 }
 
-const PostsWrap = ({ textToSearch } :PostWrapProps): JSX.Element => {
+const PostsWrap = ({  } :PostWrapProps): JSX.Element => {
```

Comment:
I see that you deleted textToSearch, is this really a change you want, I still see that we use textToSearch as props

Reply:
We are no longer receiving the "textToSearch" as a prop from the parent component, instead, "textToSearch" is now defined in the current component.

Reply:
Okay, then you can remove where we sent them from :-)